### PR TITLE
Add navbar

### PR DESCRIPTION
**Pull request by dylanerichards**
# Summary
This PR adds a navbar to the application.

- It includes a placeholder Aritzia logo, along with placeholder icons for search, favorites, and cart. 
- The "sign in" button links to the sign in page.
- The logo links to the root path of the application

Screen Shot 2022-05-16 at 12 38 15 AM

# Trello card
https://trello.com/c/nTuQ7O6s/6-add-nav-bar

# Followups
Favoriting, cart, and search will be implemented in subsequent PRs

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -5,16 +5,17 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-    <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
```
Comment (dylanerichards) [inline app/views/layouts/application.html.erb:8]: This line gives my local environment issues. I didn't mean to commit this change, but I don't think it hurts for now.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -5,16 +5,17 @@
...
+    <%# <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %1> %>
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
     <script src="https://cdn.tailwindcss.com"></script>
   </head>
 
   <body>
-    <main class="container mx-auto mt-28 flex">
```
Comment (dylanerichards) [inline app/views/layouts/application.html.erb:16]: Removing some classes from the container. I suspect they are artifacts from a tutorial that was followed initially.

File: app/views/products/index.html.erb [hunk 3]
```diff
@@ -2,7 +2,7 @@
   <% @products.each do |product| %>
     <div class="flex-none mb-5">
       <div>
-        <%= image_tag(product.image, class: "product-image mb-2") %>
+        <%= image_tag(product.image, class: "product-image mb-2", width: 390, height: 550) %>
```
Comment (dylanerichards) [inline app/views/products/index.html.erb:5]: Having to manually set the image sizes.

**Review by rjackson21 — APPROVED** [review]
LGTM! The classes removed from the container in app.html.erb were from a past tutorial and no longer useful.

**Pull request merged**